modules/setting/setting.go

```diff
@@ -944,6 +945,7 @@ func NewContext() {
 			"español", "português do Brasil", "Português de Portugal", "polski", "български",
 			"italiano", "suomi", "Türkçe", "čeština", "српски", "svenska", "한국어"}
 	}
+	DefaultLang = Cfg.Section("i18n").Key("Default").MustString("en-US")
```

Comment:
Default language should be first language from list otherwise it will be breaking change